src/chapters/errors-and-debugging/errors-and-user-input.rst

```diff
@@ -0,0 +1,108 @@
+
+========================
+Errors and User Input
+========================
```

Comment:
No need to change these on the pages already written but as you write new ones, our typical main header is indicated with just the underline ===, skipping the top line

Reply:
got it.  sometimes if i frame them, the blue boxes at the bottom of the screen appear and if i don't, then the blue boxes don't appear.  it's odd. I'll try it in chapter 8 (already started doing it in 7, so you'll see these again)